sdk/batch/azure-batch/azure/batch/_operations/_patch.py

```diff
@@ -0,0 +1,776 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Customize generated code here.
+
+Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
+"""
+import datetime
+from typing import Any, List, Optional
+from .. import models as _models
+from ._operations import (
+    BatchClientOperationsMixin as BatchClientOperationsMixinGenerated,
+)
+import collections
+import logging
+import threading
+from azure.core import MatchConditions
+from azure.core.exceptions import HttpResponseError
+from azure.core.rest import HttpResponse
+
+MAX_TASKS_PER_REQUEST = 100
+_LOGGER = logging.getLogger(__name__)
+
+__all__: List[str] = [
+    "BatchClientOperationsMixin"
+]  # Add all objects you want publicly available to users at this package level
+
+
+class BatchClientOperationsMixin(BatchClientOperationsMixinGenerated):
+    """Customize generated code"""
+
+    def create_task_collection(
+        self,
+        job_id: str,
+        task_collection: List[_models.BatchTaskCreateContent],
+        concurrencies: Optional[int] = 0,
```

Comment:
if the default is 0 can the typing simply be `int`, unless None can be passed in?

Reply:
Also, given we default it, should we make this keyword-only?

Reply:
Updated typing to be an `int` but keeping the parameter where it is since it doesn't seem like a candidate for a keyword.